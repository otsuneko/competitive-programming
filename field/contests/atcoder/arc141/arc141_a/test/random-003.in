38
569429511
400783110
290748401
420532837
750257097
49385738
57090614
868457277
806479149
976657090
950833320
552628682
16976013
78137317
20489684
464313437
531493401
174140274
2607133
156058151
97563281
58876488
286464046
346841216
843999198
771989190
818574907
100415006
808781922
96137721
461879132
437856710
330978732
857716173
796165143
846017375
641712908
267236319
