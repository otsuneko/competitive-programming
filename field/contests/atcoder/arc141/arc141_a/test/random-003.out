568568568
400400400
290290290
420420420
749749749
49384938
57085708
867867867
805805805
975975975
949949949
552552552
16971697
78127812
20482048
463463463
530530530
173173173
2222222
155155155
97559755
58875887
286286286
346346346
843843843
771771771
817817817
100100100
807807807
96129612
461461461
437437437
330330330
856856856
795795795
845845845
641641641
266266266
